.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -7,15 +7,9 @@ assignees: ''
 
 ---
 
-**Describe the issue**
-A clear and concise description of what the problem is in the spec.
+**Describe the Issue**
+Briefly describe the issue with this site. This includes all issues not related to the spec itself and includes things like the issue tracker not working, wiki pages that are incorrect, and so on.
 
-If you need help on how to use WebAudio, ask on your favorite forum or consider visiting
-the [WebAudio Slack Channel](https://web-audio.slack.com/) (register [here](https://web-audio-slackin.herokuapp.com/)) or
-[StackOverflow](https://stackoverflow.com/).
```

Comment:
We might still want to keep these.

Reply:
Well, that was a mistake. I don't know why these got changed.  Need to revert these.